### ISSUE TITLE
 Scale and orientation of the final point cloud change compared to original colmap points3D.ply

### ISSUE THREAD

**Issue by mbernier-arcturus** [nerfstudio-project/gsplat#328]
Hello, it does seem that the scale and orientation of the gaussians differ compared to original Colmap points3D.ply used for training. I was wondering if there was a feature to disable that I missed, or if there is a way to find that original transformation.

I am saving a ply file of the original gaussian using this code in simple_trainer.py (modified from #234 ):

```
def construct_list_of_attributes(self):
        l = ['x', 'y', 'z', 'nx', 'ny', 'nz']
        # All channels except the 3 DC
        if "sh0" in self.splats:
            for i in range(self.splats["sh0"].shape[1]*self.splats["sh0"].shape[2]):
                l.append('f_dc_{}'.format(i))
            for i in range(self.splats["shN"].shape[1]*self.splats["shN"].shape[2]):
                l.append('f_rest_{}'.format(i))
        else:
            for i in range(self.splats["colors"].shape[1]):
                l.append('f_dc_{}'.format(i))
            for i in range(self.splats["features"].shape[1]):
                l.append('f_rest_{}'.format(i))
        l.append('opacity')
        for i in range(self.splats["scales"].shape[1]):
            l.append('scale_{}'.format(i))
        for i in range(self.splats["quats"].shape[1]):
            l.append('rot_{}'.format(i))
        return l

    @torch.no_grad()
    def save_ply(self, path):
        os.makedirs(os.path.dirname(path), exist_ok=True)

        xyz = self.splats["means"].detach().cpu().numpy()
        normals = np.zeros_like(xyz)
        opacities = self.splats["opacities"].detach().unsqueeze(-1).cpu().numpy()
        scale = self.splats["scales"].detach().cpu().numpy()
        rotation = self.splats["quats"].detach().cpu().numpy()
        
        if "sh0" in self.splats:
            f_dc = self.splats["sh0"].detach().transpose(1, 2).flatten(start_dim=1).contiguous().cpu().numpy()
            f_rest = self.splats["shN"].detach().transpose(1, 2).flatten(start_dim=1).contiguous().cpu().numpy()
        else:
            f_dc = self.splats["colors"].detach().cpu().numpy()
            f_rest = self.splats["features"].detach().cpu().numpy()
        
        dtype_full = [(attribute, 'f4') for attribute in self.construct_list_of_attributes()]
        elements = np.empty(xyz.shape[0], dtype=dtype_full)
        attributes = np.concatenate((xyz, normals, f_dc, f_rest, opacities, scale, rotation), axis=1)
        elements[:] = list(map(tuple, attributes))
        el = PlyElement.describe(elements, 'vertex')
        PlyData([el]).write(path)
```

**Comment by liruilong940607** [comment]
Oh we by default normalize the scene for better viewing experience. The `normalize` of the colmap data parser is set to `True` in the training script:
https://github.com/nerfstudio-project/gsplat/blob/1dca18461e9dc21a301dd3fc5ad3b2aeb3fc6f20/examples/simple_trainer.py#L271

You can either disable that (set to `False`) or use the SE4 transformation matrix (original space to normalized space) to un-normalize your results:
https://github.com/nerfstudio-project/gsplat/blob/1dca18461e9dc21a301dd3fc5ad3b2aeb3fc6f20/examples/datasets/colmap.py#L193

**Comment by szhang963** [comment]
hi, how can I set this for splatfacto in nerfstudio?
I have fixed the scale_factor to 1.0, but it failed. Look at the [issue](https://github.com/nerfstudio-project/nerfstudio/issues/3319).
Thanks in advance.
